assets/company-admin/components/CompanyUsers.jsx

```diff
@@ -2,14 +2,14 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import {connect} from 'react-redux';
 
-import {gettext} from 'utils';
+import {gettext, getConfig} from 'utils';
 import {selectUser as _selectUser} from 'users/actions';
 import {currentCompanySelector, companySectionListSelector, userIdSelector, userIdMapSelector} from '../selectors';
 
 import {CompanyUserListItem} from './CompanyUserListItem';
 
 function CompanyUsersComponent({users, usersById, selectUser, activeUserId, currentCompany, companySections}) {
-    const sections = companySections[currentCompany._id];
+    const sections = getConfig('allow_companies_to_manage_products') ? companySections[currentCompany._id] : [];
```

Comment:
Rather than executing `getConfig` twice, it might be good to store the result and use that instead